plugins/Dashboard/API.php

```diff
@@ -74,8 +76,9 @@ public function createNewDashboardForUser($login, $dashboardName = '', $addDefau
         if ($addDefaultWidgets) {
             $layout = $this->dashboard->getDefaultLayout();
         }
+        $idSite = Common::getRequestVar('idSite');
```

Comment:
Same here. `Common::getRequestVar` should be avoided in API, as request parameters would automatically be mapped to the method parameters if possible.